apps/cowswap-frontend/src/modules/trade/hooks/useNotifyWidgetTrade.ts

```diff
@@ -40,7 +40,8 @@ function getTradeParamsEventPayload(tradeType: TradeType, state: TradeDerivedSta
     outputCurrencyBalance: currencyAmountToBigInt(state.outputCurrencyBalance),
     inputCurrencyFiatAmount: state.inputCurrencyFiatAmount?.toExact(),
     outputCurrencyFiatAmount: state.outputCurrencyFiatAmount?.toExact(),
-    slippageAdjustedSellAmount: currencyAmountToBigInt(state.slippageAdjustedSellAmount),
+    minimumReceiveSellAmount: currencyAmountToBigInt(state.slippageAdjustedSellAmount),
```

Comment:
In the buy order case, you'll sell more when slippage is added

```suggestion
    maximumSendSellAmount: currencyAmountToBigInt(state.slippageAdjustedSellAmount),
```

Reply:
Thank you! Renamed